automation/docker/hub-compose-fiab.yml

```diff
@@ -0,0 +1,49 @@
+version: '2'
+services:
+  hub:
+    image: selenium/hub
+    ports:
+      - "4444:4444"
+    extra_hosts:
```

Comment:
Talked about this in person. We suspect that these extra hosts are not needed here because they're supplied on the `docker run` command in `run-tests.sh`. Also `FIAB_DOMAIN` doesn't appear to be set anywhere which lends evidence to the case that these aren't needed. Either way, there is something to fix here.

If these aren't needed, then there are no remaining differences between `hub-compose.yml` and `hub-compose-fiab.yml` so they can be consolidated.